systems/plants/RigidBody.h

```diff
@@ -108,7 +108,7 @@ class DLLEXPORT_RBM RigidBody {
   MatrixXd ddTdqdq;
 
   double mass;
-  Vector4d com;  // this actually stores [com;1] (because that's what's needed in the kinematics functions)
+  Vector3d com; 
```

Comment:
Good catch.  compareParsers definitely does cause a segfault in Debug mode.  Pushing the fix now.